js/apporder.js

```diff
@@ -6,32 +6,46 @@ $(function () {
 
 	app_menu.hide();
 
-	var mapMenu = function(parent, order) {
+	var mapMenu = function(parent, order, hidden) {
 		available_apps = {};
 		parent.find('li').each(function () {
 			var id = $(this).find('a').attr('href');
+			if(hidden.includes(id)){
+				$(this).hide();
```

Comment:
Not sure if this causes issues when resizing the window. maybe removing hidden elements completely from the DOM would be better.